editor/utils/tracking.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
```

Comment:
Do we have a path forward for making this work?  (Working build system, publishing to `npm`)